app/src/main/java/com/endava/myendava/rest/RetrofitServiceApi.java

```diff
@@ -19,6 +20,9 @@
 
 public interface RetrofitServiceApi {
 
+    @GET("user/login/{email}")
+    Observable<LoginResult> userLoginMock(@Path("email")String email);
```

Comment:
can be renamed to login ... and there should be a space between @Path and the parameter. To avoid this in the future, use CTRL+ALT+L more often